orchestrator-salt/src/main/resources/salt-common/salt/monitoring/scripts/monitoring-secret-handler.sh

```diff
@@ -18,6 +18,11 @@ function main() {
   local pwd_file=$2
   local output_file=$3
   local pwd=$(cat $pwd_file | tr -d '\n')
+
+  if ! command -v htpasswd &> /dev/null; then
+    yes | yum install httpd-tools &> /dev/null
```

Comment:
usually there is no internet connection on the customers machine so this won't work. you would have to fix this on the image side
also for this we usually use salt states and not yum commands, as those are not OS independent